### ISSUE TITLE
Hyperlink element node to another diagram

### ISSUE THREAD

**Issue by lwalker-kforce** [plantuml-stdlib/C4-PlantUML#119]
Is it possible to wrap an element (e.g. `System_Boundary(...)`) to another location via hyperlink.

For example, to another diagram where the system is broken down into container or components?

I typically put my context diagram in a file `mycontext.pml` and would like to hyperlink the generated system shape (`mycontainer.pml`) such that when a user clicks on it, they are taken to the container diagram for that system, and so on.

Simply put, I would like to link my diagrams. I can wrap the entire generated diagram in a hyperlink however I would like to get more to a node level, almost akin to an image map maybe.

**Comment by adrianvlupu** [comment]
Hi,

Yes you can link elements, but it depends what is the export format of the diagram. If it's svg you can simply add the url after the boundary

```puml
Enterprise_Boundary(c0, "Widgets Limited") [[http://google.com]] {
    Person(csa, "Customer Service Agent", "Deals with customer enquiries.")
```

or after the node 

```puml
System_Ext(mail_system, "E-mail system", "The internal Microsoft Exchange e-mail system.") [[http://google.com]]
```

**Comment by huyz** [comment]
Anyone know how to make a link open in a new browser tab?
